Creating test database for alias 'default' ('file:memorydb_default?mode=memory&cache=shared')...
Operations to perform:
  Synchronize unmigrated apps: messages, rest_framework, staticfiles
  Apply all migrations: admin, auth, authtoken, contenttypes, kmdb_app, sessions
Synchronizing apps without migrations:
  Creating tables...
    Running deferred SQL...
Running migrations:
  Applying contenttypes.0001_initial... OK
  Applying contenttypes.0002_remove_content_type_name... OK
  Applying auth.0001_initial... OK
  Applying auth.0002_alter_permission_name_max_length... OK
  Applying auth.0003_alter_user_email_max_length... OK
  Applying auth.0004_alter_user_username_opts... OK
  Applying auth.0005_alter_user_last_login_null... OK
  Applying auth.0006_require_contenttypes_0002... OK
  Applying auth.0007_alter_validators_add_error_messages... OK
  Applying auth.0008_alter_user_username_max_length... OK
  Applying auth.0009_alter_user_last_name_max_length... OK
  Applying auth.0010_alter_group_name_max_length... OK
  Applying auth.0011_update_proxy_permissions... OK
  Applying auth.0012_alter_user_first_name_max_length... OK
  Applying kmdb_app.0001_initial... OK
  Applying admin.0001_initial... OK
  Applying admin.0002_logentry_remove_auto_add... OK
  Applying admin.0003_logentry_add_action_flag_choices... OK
  Applying authtoken.0001_initial... OK
  Applying authtoken.0002_auto_20160226_1747... OK
  Applying authtoken.0003_tokenproxy... OK
  Applying sessions.0001_initial... OK
System check identified no issues (0 silenced).
test_create_and_login_for_admin_account (kmdb_app.tests.TestAccountView) ... ok
test_create_and_login_for_critic_account (kmdb_app.tests.TestAccountView) ... ok
test_create_and_login_for_user_account (kmdb_app.tests.TestAccountView) ... ok
test_create_user_already_exists (kmdb_app.tests.TestAccountView) ... ok
test_wrong_credentials_do_not_login (kmdb_app.tests.TestAccountView) ... ok
test_admin_or_critic_cannot_create_comment (kmdb_app.tests.TestCommentReviewView) ... ok
test_create_comment_with_invalid_movie_id (kmdb_app.tests.TestCommentReviewView) ... ok
test_user_can_change_a_comment_on_the_movie (kmdb_app.tests.TestCommentReviewView) ... ok
test_user_can_create_comment (kmdb_app.tests.TestCommentReviewView) ... ok
test_user_cannot_change_a_comment_on_the_movie_that_did_not_comment (kmdb_app.tests.TestCommentReviewView) ... ok
test_create_review (kmdb_app.tests.TestCriticismReviewView) ... ok
test_update_review (kmdb_app.tests.TestCriticismReviewView) ... ok
test_admin_can_delete_movie (kmdb_app.tests.TestMovieRetrieveDestroyView) ... ok
test_anonymous_can_filter_movies (kmdb_app.tests.TestMovieRetrieveDestroyView) ... ok
test_anonymous_cannot_delete_movies (kmdb_app.tests.TestMovieRetrieveDestroyView) ... ok
test_anonymous_cannot_filter_movies_with_the_invalid_movie_id (kmdb_app.tests.TestMovieRetrieveDestroyView) ... ok
test_user_or_critic_cannot_delete_movies (kmdb_app.tests.TestMovieRetrieveDestroyView) ... ok
test_admin_can_create_movie (kmdb_app.tests.TestMovieView) ... ok
test_critic_or_user_cannot_create_movie (kmdb_app.tests.TestMovieView) ... ok
test_filter_movies_with_the_filter_request (kmdb_app.tests.TestMovieView) ... FAIL
test_genre_or_classification_cannot_repet (kmdb_app.tests.TestMovieView) ... ok
test_output_format_data (kmdb_app.tests.TestMovieView) ... ok

======================================================================
FAIL: test_filter_movies_with_the_filter_request (kmdb_app.tests.TestMovieView)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/felipe/projetos-kenzie/Q4/sprint2B/kmdb/kmdb_app/tests.py", line 199, in test_filter_movies_with_the_filter_request
    self.assertEqual(len(filter_movies.json()), 2)
AssertionError: 3 != 2

----------------------------------------------------------------------
Ran 22 tests in 5.397s

FAILED (failures=1)
Destroying test database for alias 'default' ('file:memorydb_default?mode=memory&cache=shared')...
